9.2/run-postgresql-master.sh

```diff
@@ -2,27 +2,16 @@
 
 set -eu
 
-source ${HOME}/common.sh
+source $HOME/common.sh
```

Comment:
Thanks for pointing that out. I didn't know we followed anything specific like that.

> In order of precedence: Stay consistent with what you find; ...; prefer "${var}" over "$var"

In multiple places in the same file and group of files we're using a mix of both styles. @mfojtik @bparees what do you think about a follow up making those consistent?!

One way or another is fine for me, as long as we keep ourselves consistent.